silq/parameters/acquisition_parameters.py

```diff
@@ -2659,3 +2661,105 @@ def analyse(self, traces=None, plot=False):
 
         self.results = results
         return results
+
+class NMRCPMGParameter(NMRParameter):
+    """ Parameter for most measurements involving an NMR pulse.
+
+    This parameter can apply several NMR pulses, and also measure several ESR
+    frequencies. It uses the `NMRPulseSequence`, which will generate a pulse
+    sequence from settings (see parameters below).
```

Comment:
Can you update this to reflect the NMRCPMGPulseSequence? Shouldn't take much, but do mention dynamical decoupling pulses somewhere.